demo/mobile/src/keycloak/authInterceptor.js

```diff
@@ -0,0 +1,30 @@
+var q;
+var auth;
+
+function AuthInterceptor($q, Auth) {
+    q = $q;
+    auth = Auth;
+}
+
+AuthInterceptor.prototype.request = function request(config) {
+  var deferred = q.defer();
+  if (auth.token) {
+    auth.updateToken(5).success(function() {
+        config.headers = config.headers || {};
+        config.headers.Authorization = 'Bearer ' + auth.token;
+
+        deferred.resolve(config);
+    }).error(function() {
+        deferred.reject('Failed to refresh token');
+    });
+  }
+  return deferred.promise;
+}
+
+angular.module('wfm-mobile').service('authInterceptor', function($q, Auth) {
+  return new AuthInterceptor($q, Auth);
+});
+
+angular.module('wfm-mobile').config(['$httpProvider', function($httpProvider) {
```

Comment:
This means that all networking should be done using angular http in order to work with keycloak.
That will be really important for documentation. We will need to change sync to cover this - provide separate cloud handler: https://github.com/feedhenry/fh-sync-js/blob/master/src/cloudHandler.js

This maybe little bit out of scope of this PR but we need to have it as ticket!